operator/controllers/syncer/csi_node.go

```diff
@@ -96,6 +95,7 @@ func GetCSIDaemonsetSyncer(c client.Client, scheme *runtime.Scheme, driver *csis
 
 	UUID = CGPrefix
 	maxUnavailable := envVars[config.DaemonSetUpgradeMaxUnavailableKey]
+	hostNetwork := envVars[config.HostNetworkKey]
```

Comment:
we don't want to set HOST_NETWORK in driver env vars, just setting it in spec is sufficient, so please add another condition similar to `if k != config.DaemonSetUpgradeMaxUnavailableKey {` below